### DISCUSSION TITLE
Craft 5 compatibility

### DISCUSSION THREAD

**Discussion by AntoineLeeroy** [sebastian-lenz/craft-linkfield#281] [General]
Hi, I was wondering if it was planned to support Craft 5 eventually.

**Comment by RyanRoberts** [comment]
Doubtful, it's now tagged as no longer maintained https://plugins.craftcms.com/typedlinkfield

**Comment by jmhall-vabeach** [comment]
That's a shame. Such a good and effective plugin.

**Comment by sebastian-lenz** [comment]
Yeah, sorry, I just don't have enough free time so support this plugin. Also in the meantime some powerful alternatives have emerged in the marketplace. So I think it makes more sense that people move on those more feature-rich alternatives and we close on this project.

**Comment by LarsDol** [comment]
Hi @sebastian-lenz, such a pity! Great plugin 🔥

Which of the alternatives would you recommend as a more-or-less drop in replacement for your linkfield?

I'm mostly worried about data migration, as many of our clients rely on this field on 1000s of pages. It would be a hard sell to tell them to go over them one by one, haha.

If anyone has written a migration to an alternative package (with Craft 5 support), we'd be very interested 🙏

**Comment by sebastian-lenz** [comment]
As far as I know Hyper (https://plugins.craftcms.com/hyper) is a more powerful variant and offers a migration.

Please feel free to share other alternatives and migration experiences!

**Comment by vandres** [comment]
@sebastian-lenz 

I total naivelly forked you repositories `craft-linkfield` and `craft-utils` locally and just changed the composer.json to allow Craft 5. Everythings seems to be working. My usage of the field is very limited though and was only used in Craft 4.

Is the update maybe just as easy as changing the composer.json?

**Reply by sgtpenguin** [reply]
I too am interested in making this plugin work for craft 5 in the interest of upgrading existing projects that depend on the field, rather than migrating to a new plugin.  @sebastian-lenz totally understand you don't have the time for this project anymore, but sounds like @vandres has had some initial success.  Would you be open to turning the project over?  I am willing to participate in making it happen.

In any event, thanks for all the work you've done thus far.  It's a great plugin!

**Reply by ammannbe** [reply]
@vandres @sebastian-lenz we would also be interested that this project stays active. At least it will become compatible with Craft CMS 5 and gets security fixes. At the moment it is working for us in Craft CMS 4 perfectly.
IMO a forking is a possibility but it would be much better for all users if we could keep this repository active.

Would you be interested in overhanding the ownership of this repository?
I would have time to keep this plugin up-to-date (especially since the upgrade to Craft CMS 5 should be very easy).

**Comment by gjhead** [comment]
Looks like a native link field is [coming to Craft 5 very soon](https://github.com/craftcms/cms/pull/15251).
